app/models/round.rb

```diff
@@ -0,0 +1,38 @@
+class Round < ApplicationRecord
+  before_create :set_position
+  before_destroy :update_positions
+
+  validates :round_type, presence: true
+  validates :round_type, uniqueness: { scope: :game_player_id,
+                                       message: 'Round already played!' }
+
+  belongs_to :game_player
+  has_one :game, through: :game_player
+
+  enum round_type: {
+    rentz_minus: 'Rentz -',
+    rentz_plus: 'Rentz +',
+    totale_minus: 'Totale -',
+    totale_plus: 'Totale +',
+    king: 'King of hearts',
+    ten: 'Ten of clubs',
+    queens: 'Queens',
+    diamonds: 'Diamonds'
+  }
+
+  def decrement_position
```

Comment:
This can also be private 🤔

Reply:
Already tried, I get same `(NoMethodError)`